install/helm/agones/templates/crds/_fleetautoscalerpolicy.yaml

```diff
@@ -0,0 +1,237 @@
+policy:
+  type: object
+  required:
+    - type
+  properties:
+    type:
+      type: string
+      enum:
+      - Buffer
+      - Webhook
+      - Counter
+      - List
+      - Chain
+    buffer:
+      type: object
+      nullable: true
+      required:
+        - maxReplicas
+      properties:
+        minReplicas:
+          type: integer
+          minimum: 0
+        maxReplicas:
+          type: integer
+          minimum: 1
+        bufferSize:
+          x-kubernetes-int-or-string: true
+          anyOf:
+            - type: integer
+            - type: string
+    webhook:
+      type: object
+      nullable: true
+      properties:
+        url:
+          type: string
+        service:
+          type: object
+          required:
+            - namespace
+            - name
+          properties:
+            namespace:
+              type: string
+            name:
+              type: string
+            path:
+              type: string
+            port:
+              type: integer
+        caBundle:
+          type: string
+          format: byte
+    counter:
+      type: object
+      nullable: true
+      required:
+        - key
+        - bufferSize
+        - maxCapacity
+      properties:
+        key:  # The name of the Counter.
+          type: string
+        minCapacity:  # Minimum aggregate counter capacity that can be provided by this FleetAutoscaler. If not specified, the actual minimum capacity will be bufferSize.
+          type: integer
+          minimum: 0
+        maxCapacity:  # Maximum aggregate counter capacity that can be provided by this FleetAutoscaler. Required.
+          type: integer
+          minimum: 1
+        bufferSize:  # Size of a buffer of counted items that are available in the Fleet (available capacity). It can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%).
+          x-kubernetes-int-or-string: true
+          anyOf:
+            - type: integer
+            - type: string
+    list:
+      type: object
+      nullable: true
+      required:
+        - key
+        - bufferSize
+        - maxCapacity
+      properties:
+        key:  # The name of the List.
+          type: string
+        minCapacity:  # Minimum aggregate list capacity that can be provided by this FleetAutoscaler. If not specified, the actual minimum capacity will be bufferSize.
+          type: integer
+          minimum: 0
+        maxCapacity:  # Maximum aggregate list capacity that can be provided by this FleetAutoscaler. Required.
+          type: integer
+          minimum: 1
+        bufferSize:  # Size of a buffer based on the list capacity that is available over the current aggregate list length in the Fleet. It can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%).
+          x-kubernetes-int-or-string: true
+          anyOf:
+            - type: integer
+            - type: string
+    {{- $featureGates := include "agones.featureGates" . | fromYaml }}
```

Comment:
(non blocking, just curious)  We've not actually done this before 🤔  - @zmerlynn is this for in-place Agones updates?

Reply:
We've done the helm-based feature gates before - do you mean we haven't done the conditional-CRD piece?

I thought it might be a good idea - then we can keep things under the feature gate until the API is "locked". Do you think it's weird?

Reply:
> - do you mean we haven't done the conditional-CRD piece?

Yep that. It's not a bad idea - can't think of anything pro/con 😄 

It does automatically mean that data is removed from the CRD if you turn off the feature flag, which is not a bad thing, and I believe is inline with what @igooch is designing, no?